src/main/java/org/pac4j/jax/rs/resteasy/Pac4JProfileInjectorFactory.java

```diff
@@ -0,0 +1,79 @@
+package org.pac4j.jax.rs.resteasy;
```

Comment:
Move this class into org.pac4j.jax.rs.resteasy.features, it would make more sense with the organization of the rest of this library :)

Reply:
✓